atlas_densities/app/data/metadata/ca1_metadata.json

```diff
@@ -1,10 +1,15 @@
 {
-    "regions": [
-        {
-            "name": "CA1",
-            "query": "CA1",
-            "attribute": "acronym",
-            "with_descendants": true
-        }
-    ]
-}
\ No newline at end of file
+    "region": {
```

Comment:
can we fix this in #34, so that that diff is easier to review, and this one is too?